src/utils/cloudinary.js

```diff
@@ -34,7 +34,7 @@ const setError = (that, res) => {
 };
 
 const setVideoSrc = (that, srcs) => {
-  console.log('Trying urls: ' + JSON.stringify(srcs));
+  console.log('Trying sources: ', srcs);
```

Comment:
I see it was previously there but can we get rid of it or hide it behind some debug mode?

Reply:
I'd like to do that, but we don't have dev-mode yet (I plan to add it) and this is logged only on error and is now much less noisy and helps debugging, so I guess it's ok for now